etc/kubernetes/kubeflow/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - https://github.com/kubeflow/pipelines/manifests/kustomize/env/platform-agnostic?ref=0.5.1
+
```

Comment:
does this work or you have to use raw?

Reply:
this works
`-k, --kustomize='': Process a kustomization directory. This flag can't be used together with -f or -R.`